cpp/oneapi/dal/algo/linear_kernel/backend/gpu/compute_kernel_dense_dpc.cpp

```diff
@@ -14,20 +14,79 @@
 * limitations under the License.
 *******************************************************************************/
 
+#define DAAL_SYCL_INTERFACE
+
 #include "oneapi/dal/algo/linear_kernel/backend/gpu/compute_kernel.hpp"
+#include "oneapi/dal/backend/interop/common.hpp"
+#include "oneapi/dal/backend/interop/common_dpc.hpp"
+#include "oneapi/dal/backend/interop/table_conversion.hpp"
+
+#include <daal/src/algorithms/kernel_function/oneapi/kernel_function_linear_dense_default_kernel_oneapi.h>
 
 namespace oneapi::dal::linear_kernel::backend {
 
+using dal::backend::context_gpu;
+
+namespace daal_linear_kernel = daal::algorithms::kernel_function::linear;
+namespace interop            = dal::backend::interop;
+
+template <typename Float>
+using daal_linear_kernel_t =
+    daal_linear_kernel::internal::KernelImplLinearOneAPI<daal_linear_kernel::defaultDense, Float>;
+
+template <typename Float>
+static compute_result call_daal_kernel(const context_gpu& ctx,
```

Comment:
Why it's `context_gpu` and not a `data_parallel_policy`?

Reply:
For consistency with cpu backend. Why not?